File: src/main/java/user11681/limitless/asm/access/EnchantmentAccess.java
package user11681.limitless.asm.access;

public interface EnchantmentAccess {
    void limitless_setMaxLevel(int level);

    int limitless_getOriginalMaxLevel();

    void limitless_setUseGlobalMaxLevel(boolean useGlobalMaxLevel);

    boolean limitless_useGlobalMaxLevel();

    int limitless_getOriginalMaxPower(int level);
}

File: src/net/auoeke/limitless/asm/access/EnchantmentAccess.kt
package net.auoeke.limitless.asm.access

@Suppress("FunctionName")
interface EnchantmentAccess {
    fun limitless_setMaxLevel(level: Int)

    fun limitless_getOriginalMaxLevel(): Int

    fun limitless_setUseGlobalMaxLevel(useGlobalMaxLevel: Boolean)

    // fun limitless_useGlobalMaxLevel(): Boolean

    fun limitless_getOriginalMaxPower(level: Int): Int
}
